proto/fs_proto.go

```diff
@@ -234,6 +236,7 @@ type TxCreateInodeRequest struct {
 	Target      []byte           `json:"tgt"`
 	QuotaIds    []uint32         `json:"qids"`
 	TxInfo      *TransactionInfo `json:"tx"`
+	FullPath    string           `json:"fullPath"`
```

Comment:
you can add a new struct for every request, like `type extend struct {fullpath string, paths []string}`, so when you  need add new extend args, it's no need update every req again, just update extend struct enough.

Reply:
Ok, I refactor it later.

Reply:
Hello @Victor1319, I already refactor it to `RequestExtend`, please review again.

Reply:
hello, there are some conflict, please fix it, and start a new pr.

Reply:
> hello, there are some conflict, please fix it, and start a new pr.

I will rebase it later.